### ISSUE TITLE
Old documentation needs to removed

### ISSUE THREAD

**Issue by trishankkarthik** [theupdateframework/tuf#164]
There are some old documentation that are no longer relevant to the current TUF specification. We should remove them, but only after copying over the relevant bits.
- [tuf/docs/tuf-client-spec.pdf](https://github.com/theupdateframework/tuf/blob/56a4a44785f34d723c34ec02669960c0b2c7c5a6/docs/tuf-client-spec.pdf)
- [tuf/docs/tuf-server-spec.pdf](https://github.com/theupdateframework/tuf/blob/56a4a44785f34d723c34ec02669960c0b2c7c5a6/docs/tuf-server-spec.pdf)
